auto-dark.el

```diff
@@ -140,8 +140,8 @@ HKCU:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize \
   "Use Emacs built-in Windows Registry function.
 In order to determine if dark theme is enabled."
   (eq 0 (w32-read-registry 'HKCU
-			   "Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize"
-			   "AppsUseLightTheme")))
+                           "Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize"
```

Comment:
If you could please undo the formatting of this hunk, as I have no windows machine right now I could test it, I'd prefer not to 'create problems'.

Also blaming the right person for this would be better :)